### DISCUSSION TITLE
Develop official Go lang client (driver)

### DISCUSSION THREAD

**Discussion by berksafran** [supabase/supabase#3434] [Feature Requests]
Hey folks,

Go lang is being popular day by day around the world. So, I can start a project which is the client (driver) for Golang developers in order to use on their projects. It can be started by cloning the JS version using Golang.

Also, If there is already a team, I can join it.

**Comment by darora** [comment]
Hey @berksafran - we have a community-maintained client at https://github.com/supabase-community/postgrest-go , feel free to contribute!

**Reply by berksafran** [reply]
Hey @darora, thanks for sharing the repository. I guess it's all about only postgrest side. I can create a new repository for "realtime". What do you think?

**Reply by hardiksachan** [reply]
Hey @berksafran, are you working on `realtime-go`? I'd love to contribute.

**Reply by snassr** [reply]
Hey guys (@berksafran) just started work on realtime-go here https://github.com/overseedio/realtime-go, let me know if you want to work together on it!

Also started a discussion here to see how we can move it to supabase-community https://github.com/supabase/supabase/discussions/5889

**Reply by snassr** [reply]
Ping me on discord if you're up for it: snassr#7594
